### PR TITLE
(2103) Simplify nation selection

### PR DESCRIPTION
**Pull request by pezholio**
This updates the nation selection part of the Profession edit journey to allow users to select United Kingdom if a profession is regulated across the UK, or "Certain Nations" if the profession is only regulated in particular nation(s)

# Screenshots

image

image

### CODE REVIEW

File: src/professions/admin/top-level-information.controller.ts [hunk 1]
```diff
@@ -149,6 +151,7 @@ export class TopLevelInformationController {
   private async renderForm(
     res: Response,
     name: string,
+    coversUK: string,
```
Comment (Gweaton) [inline src/professions/admin/top-level-information.controller.ts:164]: Does it make sense to have this as boolean here? We can convert it like:`Boolean(Number(submittedValues.coversUK))`

**Review by Gweaton — APPROVED** [review]
This looks good - just had one comment on the use of `1` and `0` for our values here. It's hard to convert checkboxes to booleans, but I feel it's a bit clearer to pass round boolean value as much as possible where we can.

File: src/professions/admin/top-level-information.controller.ts (hunk 1 above)
Comment (Gweaton) [reply src/professions/admin/top-level-information.controller.ts:164]: There are other places in this PR that use 1 and 0, can we replace as many of those as possible with a true or false value?

Comment (pezholio) [reply src/professions/admin/top-level-information.controller.ts:164]: Yeah, I tried that but, as you say, it was tricky to convert them (I expected the DTO to do it, but it wasn't having any of it!). I'll have a crack with your suggestion.

Comment (pezholio) [reply src/professions/admin/top-level-information.controller.ts]: That's got it - much better!

**Pull request merged**